js/src/components/supported-country-select/supported-country-select.js

```diff
@@ -22,17 +24,22 @@ import useMCCountryTreeOptions from './useMCCountryTreeOptions';
  *
  * @param {Object} props React props.
  * @param {Array<CountryCode>} [props.countryCodes] Country codes for converting to tree-based option structure. It will use all MC supported countries if not specified.
- * @param {Object} props.restProps Props to be forwarded to TreeSelectControl.
+ * @param {string} props.className Class name. Note that when there is a Form's `{...getInputProps('fieldname')}` being passed into this component, the className might be null.
```

Comment:
As stated in the JSDoc comment here, the `className` props is `null` because it was passed in by `Form`'s `{ ...getInputProps('...') }`. See: https://github.com/woocommerce/woocommerce-admin/blob/78d0e10a99676c92650a5248917207e7304f5c1b/packages/components/src/form/index.js#L130

Reply:
This comment is a bit confusing for me. Shouldn't be then `{string|null}` instead of just string and then explaining the case with getInputProps? As well as  set up the prop as optional. 

Also I don't understand why we removed  the line regarding

`* @param {Object} props.restProps Props to be forwarded to TreeSelectControl.`

Reply:
> Shouldn't be then `{string|null}` instead of just string and then explaining the case with getInputProps? 

IMO, from the perspective of `SupportedCountrySelect`, className should always be string, and I see WC Form passing className null as a bug.

> As well as set up the prop as optional.

Okay, I can make it into `[props.className]`. 

> Also I don't understand why we removed the line regarding
> 
> `* @param {Object} props.restProps Props to be forwarded to TreeSelectControl.`

Because there isn't a prop called `restProps`? 😅 

```jsx
<SupportedCountrySelect
	restProps={ {} }  // no such prop...?
/>
```

Reply:
> Okay, I can make it into `[props.className]`.

I have updated this in [`ab675ab` (#1465)](https://github.com/woocommerce/google-listings-and-ads/pull/1465/commits/ab675ab8aab972a7a62740d67f1910df5a25b9fa).

Reply:
> Because there isn't a prop called restProps? 😅

It's actually.

Check Line 50 of `js/src/components/supported-country-select/supported-country-select.js`

Reply:
> It's actually.

The `restProps` in the code is "an object containing all other props except countryCodes and className". `restProps` itself is not a prop that we can access like `props.restProps`, and we do not assign `restProps` like `<SupportedCountrySelect restProps={ {} } />`.

> I guess the attempt here was just indicating that `SupportedCountrySelect` send `...restProps` into `TreeSelect` component.

Good point. I added JSDoc about this in [`5983adc` (#1465)](https://github.com/woocommerce/google-listings-and-ads/pull/1465/commits/5983adceebb5cbcefb4f985a6b090da72e78e9d7). 🙂